metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -609,6 +609,18 @@ object ServerCommands {
     "[uri], the uri of the worksheet that you'd like to copy the contents of.",
   )
 
+  val CopyFQNOfSymbol = new ParametrizedCommand[TextDocumentPositionParams](
+    "copy-fqn",
+    "Copy fully qualified name of symbol",
+    s"""|Copy the fully qualified name of a symbol to the clipboard.
+        |
+        |Note: This command returns the fully qualified name of the symbol, and the LSP client
+        |is in charge of taking that content and putting it into your local buffer.
```

Comment:
I think it'd be cool to have a special client command for putting something in the clipboard.